src/Explorer/Tree/Collection.ts

```diff
@@ -1145,3 +1161,7 @@ export default class Collection implements ViewModels.Collection {
     }
   }
 }
+
+function sleep(seconds: number) {
```

Comment:
seconds doesn't seem like the right var name for this

Reply:
possibly could move this to a util file if it exists, may be useful elsewhere in the project

Reply:
seconds actually is fine I think, I just found it confusing that we're passing `retryAttempts` as a `seconds` param without any backoff description